### PR TITLE
First draft resources page with training videos

### PR DESCRIPTION
**Pull request by kkraune**
- The videos are not findable without the link for now - will make public once a few more have had a look
- The resource page not linked from menus yet

FYI @jobergum

### CODE REVIEW

**Comment by kkraune** [comment]
... or @bratseth

File: resources.html [hunk 1]
```diff
@@ -0,0 +1,180 @@
...
+          <p><code>docker exec vespa curl -s http://localhost:19110/state/v1/health</code></p>
+          <p><code>docker exec vespa /opt/vespa/bin/vespa-logfmt -S container</code></p>
+          <p><code>ls -la target/albums-deploy.jar</code></p>
+        </div>
+
+
+
+        <h2 id="troubleshooting-startup-multinode">Troubleshooting startup - multinode</h2>
+        <ul>
+          <li>How to start a Vespa, using multiple Config Servers.</li>
```
Comment (bratseth) [inline resources.html:78]:
```suggestion
          <li>How to start a Vespa instance with multiple config servers.</li>
```

File: resources.html [hunk 2]
```diff
@@ -0,0 +1,180 @@
...
+          <li>Validate that all servers are started.</li>
+          <li>How Vespa Config Servers use Apache Zookeeper, and check for quorum / leader election.</li>
```
Comment (bratseth) [inline resources.html:80]:
```suggestion
          <li>How Vespa config servers use Apache ZooKeeper, and check for quorum / leader election.</li>
```

**Review by bratseth — APPROVED** [review]
Great!

**Pull request merged**